.ci/build_nccl.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -leE
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
not needed
if needed below - can scriptdir=topdir/scripts (topdir from settings.sh)

Reply:
SCRIPT_DIR is needed to correctly source settings.sh by absolute paths - verified with main script called from folders on different levels and even when script is called with relative paths.